inst/include/dust/dust.hpp

```diff
@@ -139,30 +139,38 @@ class Dust {
   }
 
   void run(const size_t step_end) {
+#ifdef _OPENMP
```

Comment:
Here is a controversial thought: if you don't have an openmp compiler, how much is gained by using the package? Maybe a big fat warning when it doesn't understand the pragma would inspire a user to check the docs and try and enable it

Reply:
I don't have an openmp compiler on my laptop 😁 

The warnings would prevent other people writing models and getting them on CRAN, for example. Someone should be able to run sircovid on their non-openmp-enabled computer and get the same answer